_pages/release_notes.md

```diff
@@ -172,7 +172,7 @@ The additional APIs that have been added in this release are:
 
 <img src="{{site.cdnurl}}{{site.baseurl}}/images/post_images/release_notes/19.01_release_notes_algorithm_actions.png" alt="Release Notes algorithmia management APIs" class="screenshot">
 
-These algorithm APIs do not require management API keys to use. Non-administrator users of an Algorithmia platform will be able to use their [simple API keys]({{site.baseurl}}/basics/customizing-api-keys) to interact with algorithms they have permissions to work with. The use of Management API keys will not be supported
+These algorithm APIs do not require admin API keys to use. Non-administrator users of an Algorithmia platform will be able to use their [simple API keys]({{site.baseurl}}/basics/customizing-api-keys) to interact with algorithms they have permissions to work with. The use of admin API keys will not be supported
```

Comment:
Minor: let's remove `simple` here.

Reply:
This will be removed once the other PR is merged. 👍